### PR TITLE
feat(#961): Rules metrics in sidebar

### LINKED ISSUE #961: [Weak supervision] Rules numbers by label
For instance:

Sci/tech 2
Sports 1
Business 4
Politics 0
World 0

his feature could be used for two things:
- Help to know how is going the rule definition 
- See the full label list (in "define rules" we dont have this list by default)

### PR DESCRIPTION
**Pull request by leiyre**
closes #961

This PR includes rule metrics in sidebar and some changes in sidebar styles and logic

### CODE REVIEW

File: frontend/components/commons/sidebar/SidebarPanel.vue [hunk 1]
```diff
@@ -16,12 +16,14 @@
   -->
 
 <template>
-  <aside
-    :style="{ top: topPosition }"
-    :class="['sidebar', annotationEnabled ? 'annotation' : 'explore']"
-  >
+  <aside :class="['sidebar', annotationEnabled ? 'annotation' : 'explore']">
```
Comment (frascuchon) [inline frontend/components/commons/sidebar/SidebarPanel.vue]: Define rule and explore shares class ?

Comment (frascuchon) [inline frontend/components/commons/sidebar/SidebarPanel.vue]: Define rule and explore shares class ?

Comment (leiyre) [reply frontend/components/commons/sidebar/SidebarPanel.vue]: @frascuchon we can remove mode reference with the new header

**Comment by frascuchon** [comment]
why there are so many affected files? Is there a way to centralize styles config and avoid all CSS changes? 

cc @leiyre

**Comment by leiyre** [comment]
@frascuchon there are so many changes because I centralised some sidebar settings as a variable to avoid so many changes in the future

**Review by frascuchon — APPROVED** [review]

**Pull request merged**